provisioner/data.js

```diff
@@ -529,6 +521,10 @@ WorkerType.prototype.determineSpotBids = function(regions, pricing, runningCapac
   
   var change = this.determineCapacityChange(runningCapacity, pendingCapacity, pending);
 
+  // TODO: If change is negative we should cancel spot requests... This should
+  // go through the aws-manager. IMO this method belongs in the provision.js
+  // file...
```

Comment:
The old provisioner does it here:
https://github.com/taskcluster/aws-provisioner/blob/master/provisioner/provision.js#L19-L56

Decide how many to cancel and how many to request... The `chance` returned from `determineCapacityChange` in this PR is already corrected for max/min...